spec/spec_helper.rb

```diff
@@ -1,6 +1,8 @@
 require 'sad_panda/version'
 require './lib/sad_panda'
 require 'lingua/stemmer'
-require './lib/sad_panda/emotions/emotion_bank'
-require './lib/sad_panda/emotions/term_polarities'
-require './lib/sad_panda/emotions/stopwords'
+require './lib/sad_panda/bank/emotions'
+require './lib/sad_panda/bank/polarities'
+require './lib/sad_panda/bank/stopwords'
+
+require 'pry'
```

Comment:
@EdwinRozario we probably don't need `pry` outside of the debugging/development stage. Let's remove it here

Reply:
Done!